client/src/pages/students/player.jsx

```
@@ -1,10 +1,457 @@
+                <motion.button
+                  className="bg-gradient-to-r from-blue-500 to-purple-600 text-white px-6 py-2 rounded-full font-bold shadow-lg hover:shadow-xl transition-all duration-300"
+                  whileHover={{
+                    scale: 1.05,
+                    boxShadow: "0 12px 24px rgba(59, 130, 246, 0.4)",
+                  }}
+                  whileTap={{ scale: 0.95 }}
+                  initial={{ opacity: 0, x: 20 }}
+                  animate={{ opacity: 1, x: 0 }}
+                  transition={{ delay: 0.4 }}
+                >
+                  {false ? "Completed ✓" : "Mark Complete"}
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)